### DISCUSSION TITLE
GitHub Galaxy Brain

### DISCUSSION THREAD

**Discussion by abubakaristiak** [recodehive/Stackoverflow-Analysis#521] [Q&A] [answered]
Which of the following GitHub actions is used to submit changes to a repository from your local machine?

A) Fork
B) Clone
C) Commit
D) Push

**Comment by mullarohan** [comment, marked as answer]
D) Push

**Comment by OrasanuAna** [comment]
The correct answer is D) Push.

Here’s why:  
- Fork is used to create a copy of someone else's repository in your GitHub account.  
- Clone is for downloading a repository to your local machine.  
- Commit saves your changes locally, but doesn’t send them to GitHub.  
- Push is the action that actually submits your committed changes from your local machine to the repository on GitHub.  

So, when you want to submit changes, you use Push.
